app/queue/event_producer.py

```diff
@@ -41,6 +43,11 @@ def __init__(self, config, topic):
         self.mq_topic = topic
 
     def write_event(self, event, key, headers, *, wait=False):
+        # This is an adhoc for us to test kessel migration on the new OCP replica namespace
+        REPLICA_NAMESPACE = os.environ.get("REPLICA_NAMESPACE", "false").lower() == "true"
+        if REPLICA_NAMESPACE:
+            return
+
```

Comment:
If you set the var on `config.py`, you can then reference it here without having to evaluate it each time. Small perf increase, but `write_event()` is called a whole lot.
(Also, I think it's good to try to keep our env var checks in `config.py` for the most part, just to have them all in one place)
```suggestion
        self.mq_topic = topic
        self.bypass_write_event = config.replica_namespace

    def write_event(self, event, key, headers, *, wait=False):
        # This is an adhoc for us to test kessel migration on the new OCP replica namespace
        if self.bypass_write_event:
            return

```